src/database/mod.rs

```diff
@@ -619,13 +631,21 @@ pub mod test {
         let res = db.del_last_index(keychain).unwrap().unwrap();
 
         assert_eq!(res, 1);
+
+        let res = db.get_last_index(keychain).unwrap();
+        assert!(res.is_none());
     }
 
     pub fn test_check_descriptor_checksum<D: Database>(mut db: D) {
+        // insert checksum associated to keychain
         let checksum = "1cead456".as_bytes();
         let keychain = KeychainKind::External;
         let _res = db.check_descriptor_checksum(keychain, checksum);
 
+        // check if `check_descriptor_checksum` throws
```

Comment:
Those comments are great, but included in the wrong commit - should be in fd48469b617b2a0335d8a982d913434e8b6df77e, not 2ea1f574928b85ee00829efa9cfc4b23e8a712df